pkg/verifier/notation/notation.go

```diff
@@ -181,7 +188,39 @@ func getVerifierService(conf *NotationPluginVerifierConfig, pluginDirectory stri
 	if err != nil {
 		return nil, err
 	}
-	verifier, err := notationVerifier.New(&conf.TrustPolicyDoc, store, NewRatifyPluginManager(pluginDirectory))
+
+	// revocation check using corecrl from notation-core-go and crl from notation-go only
+	crlFetcher, err := corecrl.NewHTTPFetcher(&http.Client{Timeout: 5 * time.Second}) // Todo: replace with crlprovider notation implementation function
```

Comment:
would you implement the crlprovider in a different PR?

Reply:
Sure

Reply:
Have `crlprovider` renamed to `revocationfactory` and implemented in PR: https://github.com/ratify-project/ratify/pull/1890